redisson/src/main/java/org/redisson/api/RedissonRxClient.java

```diff
@@ -15,6 +15,7 @@
  */
 package org.redisson.api;
 
+import org.checkerframework.checker.units.qual.K;
```

Comment:
this dependency should be removed

Reply:
Very sharp, this was unintended, for some reason my IDE added this. Removed it now.